Sidebar.jsx

```diff
@@ -1,22 +1,38 @@
 import styles from "./Sidebar.module.scss"
 import React from 'react'
-import { Link} from '@tanstack/react-location'
+import { Link,useLocation, useRouter } from '@tanstack/react-location'
+import { useEffect } from "react";
```

Comment:
Here aswell, we already imported React, so it would be better to just remove this individual import and just do `React.useEffect`.